libs/components/src/lib/dial-pad/dial-pad.ts

```diff
@@ -163,40 +163,26 @@ export class DialPad extends FoundationElement {
 	 */
 	_deleteLastCharacter = () => {
 		this.value = this.value.slice(0, -1);
-	};
-
-	/**
-	 *
-	 * @internal
-	 */
-	_handleInput = () => {
-		this.value = this._textFieldEl.value;
 		this.$emit('input');
-	};
-
-	/**
-	 *
-	 * @internal
-	 */
-	_handleChange = () => {
-		this.value = this._textFieldEl.value;
 		this.$emit('change');
+		if (this.value === '') {
+			document.addEventListener(
+				'blur',
+				this.#blurHandlerAfterDeleteButtonRemoved,
+				true
+			);
```

Comment:
Why would this be needed?
If there is actually a reason for it needs to be explained in a comment

```suggestion
			this._textFieldEl.focus();
```

Reply:
1. We need to prevent the focus event from firing
2. This is tested
3. I've wrapped the logic in a function with a descriptive name.

Reply:
I don't understand. The component will not emit focus (or blur) when focus moves within, so this should not be necessary.

Reply:
See the related issue.
When pressing the delete button (in the UI) to delete the last character in the input, it fires a blur event and the dial pad loses focus. This makes sure it remains that way. 
See the descriptions:
<img width="679" alt="image" src="https://github.com/user-attachments/assets/e0530dcc-af70-4b32-9a45-8a249f30fc16">

Reply:
This solves the problem of focus leaving the component after deleting the last character. Focus does not remain inside the component hence the event being fired.

Reply:
Yes I understand that. But how does the suggested change not resolve the issue? Focus moves to text-field, therefore no blur (or focus) will be emitted on the dial-pad

Reply:
Oh, I see what you mean. 
The focus will happen before the event dispatches.
Nice one, thanks!